devices/google-blueline/firmware/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, runCommandNoCC
+, fetchFromGitHub
+, fetchurl
+, firmwareLinuxNonfree
+}:
+
+let
+  # The following files, though required, are not present in the modem
+  # partition.
+  cfg = fetchurl {
+    url = "https://raw.githubusercontent.com/LineageOS/android_device_motorola_addison/f99c3591c83c19da6db096eb3f2e5fb0e0d91eed/wifi/WCNSS_qcom_cfg.ini";
+    sha256 = "1dkmjm2j5l5c6a4q1xsdjkfqqy8d5aj9qd35al4lz6ma58gcy62y";
+  };
```

Comment:
Holy bat-copy-and-paste from addison, batman! :)

You'll be able to drop it and the inherit.